src/NzbDrone.Core/Queue/DatetimeComparer.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace NzbDrone.Core.Queue
 {
-    public class EstimatedCompletionTimeComparer : IComparer<DateTime?>
+    public class DatetimeComparer : IComparer<DateTime?>
```

Comment:
I renamed this file -- is this going to cause issues pulling this commit into Radarr/Lidarr/etc?

Reply:
It shouldn't be a problem at all.